### PR TITLE
fix: update gas parameters for a parachain, units, and miscellaneous fixes and cleanups 

### PR DESCRIPTION
**Pull request by peterwht**

### CODE REVIEW

**Review by AlexD10S — APPROVED** [review]

**Comment by AlexD10S** [comment]
Amazing!

**Review by AlexD10S — APPROVED** [review]
Tested after the rebase and works!

Only doubt is about the renaming of `parachain-template-runtime`to  `frontier-parachain-runtime` in the code.
I changed to `parachain-template-runtime` to keep consistency with the node binary name. The binary name has to be `parachain-template-node`, so when we generate the Zombienet config file with `pop-cli`, it works.

Just my opinion about naming, rest looks great.,
 If you keep the renaming here you can merge: https://github.com/r0gue-io/evm-parachain/pull/4 after. Otherwise just close that PR

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -359,16 +362,30 @@ impl<F: FindAuthor<u32>> FindAuthor<H160> for FindAuthorTruncated<F> {
     }
 }
 
-const BLOCK_GAS_LIMIT: u64 = 75_000_000;
+/// Current approximation of the gas/s consumption considering
+/// EVM execution over compiled WASM (on 4.4Ghz CPU).
+/// Given the 500ms Weight, from which 75% only are used for transactions,
```
Comment (Daanvdplas) [inline runtime/src/lib.rs]: This is incorrect, this is 2 seconds

**Review by Daanvdplas — APPROVED** [review]
I don't understand though why we change the names to frontier, and in #4 we change it back.

**Comment by AlexD10S** [comment]
> I don't understand though why we change the names to frontier, and in #4 we change it back.

It closed the other PR to avoid more confusion. It made sense when it was opened but not after the merge or your PR where naming was changed already: https://github.com/r0gue-io/evm-parachain/pull/11/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R32.

The binary name has to be called `parachain-template-node`, so when we generate the Zombienet config file with pop-cli, it works without changing anything.

For consistency I would prefer to keep `parachain-template-runtime` too instead of `frontier-parachain-runtime`

**Review by Daanvdplas — CHANGES_REQUESTED** [review]
Change `frontier-parachain` to `parachain-template` naming

**Review by AlexD10S — APPROVED** [review]

**Review by Daanvdplas — APPROVED** [review]
LGTM!

**Pull request merged**